activesupport/lib/active_support/core_ext/numeric/conversions.rb

```diff
@@ -117,7 +117,18 @@ def to_s(*args)
     when :human_size
       return ActiveSupport::NumberHelper.number_to_human_size(self, options)
     else
-      super
+      case self
+      when Float
+        return super()
+      when Fixnum, Bignum, BigDecimal
+        return (if format.is_a? Symbol
+          super()
+        else
+          super
+        end)
+      else
+        super
+      end
```

Comment:
This module is only prepended into `Fixnum`, `Bignum`, `Float` and `BigDecimal`, so I think we can simplify the code to:

```ruby
if is_a?(Float) || format.is_a?(Symbol)
  super()
else
  super
end
```

Though to be honest, the `is_a?` checks suggests that this might not be the right abstraction level. In other words this module tries to be independent of it's includers, but can't. @rafaelfranca (since you commented on this), do you have thoughts on what to do?